lib/community/digest2/personalizeData.js

```diff
@@ -81,9 +81,10 @@ const personalizeData = (user, data, opts = {}) => {
       name: user.get('name')
     },
     email_settings_url: Frontend.Route.userSettings() + clickthroughParams + '&expand=account',
-    tracking_pixel_url: Analytics.pixelUrl('Digest', {userId: user.id, community: data.community_name}),
-    form_action_url: Frontend.Route.emailPostForm(),
-    form_token: Email.postCreationToken(data.community_id, user.id)
+    tracking_pixel_url: Analytics.pixelUrl('Digest', {userId: user.id, community: data.community_name, versionName: opts.versionName}),
```

Comment:
can you make the key "Email Version" instead of "versionName"? The key shows up in Mixpanel exactly as it's defined here.